template.yaml

```diff
@@ -70,14 +70,39 @@ Resources:
           Properties:
             # Run every 15 minutes
             Schedule: cron(*/15 * * * ? *)
-
       Policies:
         - DynamoDBCrudPolicy:
             TableName: !Ref ManifestTable
         - AWSSecretsManagerGetSecretValuePolicy:
             SecretArn: !Ref SecretArn
 
+  RedirectorFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      MemorySize: 128
+      # These should be very quick, since we're only making one DynamoDB read request, but due to the
```

Comment:
Even though this lambda doesn't require Box, we've linked it to the dependencies for Box through the common package, adding the overhead to the cold start. We might want to think of ways to avoid this in the future, although it's probably not a very big deal.

Reply:
Let's defer this until it becomes clear that it's a problem.